portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -19,12 +19,19 @@
 
   private final String name;
   private final String likedGame;
-  private final String comment;
+  private final String content;
+  private final long timestamp;
+  private final long id;
 
-  public Comment(String name, String likedGame, String comment){
+  public Comment(long id, String name, String likedGame, String content, long timestamp){
```

Comment:
I suggest reading about Builder design pattern in Java:

https://dzone.com/articles/design-patterns-the-builder-pattern

It's optional and might be out of scope for the portfolio project, but it's good to know for future reference :)